package.json

```diff
@@ -175,18 +175,18 @@
     "@uniswap/merkle-distributor": "1.0.1",
     "@uniswap/permit2-sdk": "1.2.0",
     "@uniswap/redux-multicall": "^1.1.8",
-    "@uniswap/router-sdk": "^1.3.0",
-    "@uniswap/sdk-core": "^3.2.6",
-    "@uniswap/smart-order-router": "3.13.5",
+    "@uniswap/router-sdk": "^1.6.0",
+    "@uniswap/sdk-core": "^4.0.3",
+    "@uniswap/smart-order-router": "3.13.7",
```

Comment:
unpin this (add a `^`) for more flexibility.
same with the rest of the uniswap deps. to avoid duplication, it's easier if they're `^` deps.

Reply:
I've updated all `@uniswap` depedencies to use latest. sdk-core was the only major version bump